packages/dropdowns/src/elements/Dropdown/Dropdown.tsx

```diff
@@ -215,19 +222,6 @@ const Dropdown: React.FunctionComponent<IDropdownProps & ThemeProps<DefaultTheme
               }
 
               return changes;
-            case Downshift.stateChangeTypes.controlledPropUpdatedSelectedItem:
-            case Downshift.stateChangeTypes.mouseUp:
-            case Downshift.stateChangeTypes.keyDownSpaceButton:
-            case Downshift.stateChangeTypes.blurButton:
-            case Downshift.stateChangeTypes.blurInput:
-              return {
-                ...changes,
-                inputValue: ''
-              };
-            case Downshift.stateChangeTypes.keyDownEnter:
-            case Downshift.stateChangeTypes.clickItem:
-            case REMOVE_ITEM_STATE_TYPE as any:
-              return { ...changes, inputValue: '', isOpen: false };
```

Comment:
Avoid all unneeded `stateReducer` logic because we are moving towards the approach where the user manually clears out the input when they want, instead of us doing it for them.